src/main/java/uk/gov/justice/probation/courtcaseservice/service/OffenderUpdateService.java

```diff
@@ -36,10 +36,7 @@ public OffenderUpdateService(DefendantRepository defendantRepository, OffenderRe
     @Transactional
     public void removeDefendantOffenderAssociation(final String defendantId) {
         final var defendant = findDefendantOrElseThrow(defendantId);
-
-        if (!StringUtils.isEmpty(defendant.getCrn())) {
-            defendantRepository.save(defendant.withCrn(null).withId(null));
-        }
+        defendantRepository.save(defendant.withCrn(null).withManualUpdate(true).withOffenderConfirmed(true).withId(null));
```

Comment:
It won't actually affect this bug but we should add `.withManualUpdate(true).withOffenderConfirmed(true)` to line 65 as well so that the DB correctly reflects whether it's been confirmed/manually updated.